docs/build/javascript/quick-start.md

```diff
@@ -0,0 +1,146 @@
+    
+    const content = {
+      name: 'Your Name',
+      emoji: '✌🏻',
+      description: 'Something about yourself'
+    }
+    const metadata = {
+      controllers: [ceramic.did.id],
+      family: 'Basic Profile',
+      schema: schemaId
+    }
+
+    const doc = await TileDocument.create(ceramic, content, metadata)
+
+    const streamId = doc.id.toString()
+    ```
+
+## **3. Query a stream**
+Use the `load()` function to query the current [state](../learn/glossary.md#state) of a stream. You will need to provide it's *StreamID*.
+
+=== "Command"
+
+    ```JavaScript
+    const streamId = "kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa"
+    const stream = await TileDocument.load(streamId)
+    ```
+
+## **4. Update a stream**
+Use the `update`  command to update a stream. your [DID](../learn/glossary.md#dids) must be in the [controller](../learn/glossary.md#controllers) of the stream in order to update it. 
+
+=== "Command"
+
+    ```JavaScript
+    const streamId = "kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa"
+    const doc = await TileDocument.load(ceramic, streamId, opts)
+    
+    await doc.update(newContent, newMeta, opts)
+    ```
+
+
+In the following example we update a TileDocuments' content while also giving it a tag.
+
+=== "Command"
+
+    ```JavaScript
+    const streamId = "kjzl6cwe1jw147ww5d8pswh1hjh686mut8v1br10dar8l9a3n1wf8z38l0bg8qa"
+    const doc = await TileDocument.load(ceramic, streamId)
+
+    await doc.update({foo: 'baz'}, {tags: ['baz']})
+    ```
+<!-- API Reference here -->
+
+## **5. Create a schema**
+TileDocuments can enforce that their contents adhere to a specified schema. The schemas themselves are TileDocuments where the content is a [json-schema](https://json-schema.org){:target="_blank"}. For example we can create a schema that requires a TileDocument to have a *title* and a *message*.
+
+=== "Command"
+
+    ```JavaScript
+    const schema = {
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "title": "Reward",
+      "type": "object",
+      "properties": {
+        "title": { "type": "string" },
+        "message": { "type": "string" }
+      },
+      "required": [
+        "message",
+        "title"
+      ]
+    }
+    const metadata = {
+      controllers: [ceramic.did.id] // this will set yourself as the controller of the schema
+    }
+    const rewardSchema = await TileDocument.create(ceramic, schema, metadata)
+    ```
+
+## **6. Create a TileDocument stream that uses a schema**
+First, use the `rewardSchema.commitId.toString()` to get the current [commitIDs](../learn/glossary.md#commitid) of the schema stream. When creating a TileDocument that uses this schema need to use a commitID instead of the StreamID. This is to enforce that we are using a specific version of the schema since the schema stream is mutable and can be updated.
```

Comment:
```suggestion
First, use the `rewardSchema.commitId.toString()` to get the current [CommitID](../learn/glossary.md#commitid) of the schema stream. When creating a TileDocument that uses this schema you need to use a CommitID instead of the StreamID. This is to enforce that we are using a specific version of the schema since the schema stream is mutable and can be updated.
```